x-pack/plugins/security_solution/common/endpoint/data_generators/endpoint_action_generator.ts

```diff
@@ -178,6 +186,32 @@ export class EndpointActionGenerator extends BaseDataGenerator {
     return super.randomN(max);
   }
 
+  randomResponseActionRunningProcesses(n?: number): RunningProcessesEntry[] {
+    const numberOfEntries = n ?? this.randomChoice([0, 1, 2, 4, 8]);
```

Comment:
`8` seems low to me for "real world" cases. Doing a `top` on my machines, I have over 384 processes. I'm not saying we should generate that much data, but maybe something a little larger so that it fills up the screen (specially with the `action_resolver` script).

perhaps `[20, 30, 40, 50]`?